docs/reference/utilities/text.md

```diff
@@ -0,0 +1,247 @@
+# Typography
+
+Usual text utility classes to overcome the common text management needs.
+
+## Text Alignment
+
+<br> Is used for aligning the inner content of a block element.
+
+| Class         | Property             |
+| ------------- | ---------------------|
+| u-text-left   | `text-align: left`   |
+| u-text-center | `text-align: center` |
+| u-text-right  | `text-align: right`  |
+| u-text-justify| `text-align: justify`|
+
+### Left
+<br> Use `u-text-left` to aligns the text to the left:
+
+```html
+<p class="u-text-branding-primary u-text-left">Hello my name is Normandy!</p>
+```
+
+<pre>
+    <p class="u-text-branding-primary u-text-left">Lorem ipsum dolor sit amet, consectetuer adipiscing elit.<br>Aenean commodo ligula eget dolor.<br>Aenean massa. Cum sociis natoque penatibus et magnis dis<br>parturient montes,nascetur ridiculus mus. Donec quam felis,<br>ultricies nec, pellentesque eu, pretium quis, sem.
```

Comment:
Per fer salts de paràgrafs, millor tancar i obrir una etiqueta `<p>`. `<br>` no té la funció de fer saltar paràgrafs. (oju que passa a més llocs).

w3schools: "Note: Use the <br> tag to enter line breaks, not to separate paragraphs."